docs/external-node.md

```diff
@@ -358,26 +373,41 @@ Note: Only Windows Server 2019 is supported in the first release at the moment.
    Copy-Item .\antrea-agent.conf $WIN_AGENT_CONF_PATH
    ```
 
-3. Configure environment variable `NODE_NAME` if the VM's hostname is different
-   from the name defined in the `ExternalNode` resource.
-
-   ```powershell
-   [Environment]::SetEnvironmentVariable("NODE_NAME", "vm1")
-   [Environment]::SetEnvironmentVariable("NODE_NAME", "vm1", [System.EnvironmentVariableTarget]::Machine)
-   ```
-
-4. Create `antrea-agent` service using nssm. Below is a sample snippet to start
-   `antrea-agent` as a service:
-
-   ```powershell
-   $WIN_AGENT_BIN_PATH="C:\antrea-agent"
-   $WIN_AGENT_LOG_PATH="C:\antrea-agent\logs"
-   New-Item -ItemType Directory -Force -Path $WIN_AGENT_BIN_PATH
-   New-Item -ItemType Directory -Force -Path $WIN_AGENT_LOG_PATH
-   Copy-Item .\antrea-agent.exe $WIN_AGENT_BIN_PATH
-   nssm.exe install antrea-agent $WIN_AGENT_BIN_PATH\antrea-agent.exe --config $WIN_AGENT_CONF_PATH\antrea-agent.conf --log_file $WIN_AGENT_LOG_PATH\antrea-agent.log --logtostderr=false
-   nssm.exe start antrea-agent
-   ```
+3. Bootstrap `antrea-agent`
+
+   1. Bootstrap `antrea-agent` using the [installation script](../hack/externalnode/install-vm.ps1)
+       as shown below.
```

Comment:
```suggestion
      as shown below (only specific Windows versions have been tested and are supported).
```

Reply:
Done

Reply:
Only one version of Windows is supported. So updated it